generic/sysbench.py

```diff
@@ -64,9 +65,17 @@ def setUp(self):
                 self.urllink = self.params.get(
                     'url-link', default="https://github.com/akopytov/"
                                         "sysbench.git")
+                self.fixlink = self.params.get('fixlink', default=None)
+                self.fix_dir = self.params.get('fixdir', default=None)
                 self.bch = self.params.get('branch', default='master')
                 git.get_repo(self.urllink, branch=self.bch,
                              destination_dir=self.teststmpdir)
+
+                if self.fixlink and self.fix_dir:
+                    luajiturl = '%s%s' % (self.teststmpdir, self.fix_dir)
```

Comment:
Very minor, but I would name this variable something generic.
Let me know if you can quickly do it, else I am fine with merging this one.